extension/sidebar-pane.html

```diff
@@ -127,6 +127,53 @@
       font-weight: 600;
     }
 
+    /* occurrences part */
+    .occurrences header {
+      display: flex;
+      align-items: center;
+      cursor: pointer;
+      color: #b6babf;
+    }
+
+    .occurrences header::before {
+      content: "";
+      width: 16px;
+      height: 16px;
+      background-image: url(images/dropmarker.svg);
+      display: inline-block;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    .occurrences ul {
+      line-height: 1.2;
+      padding-inline-start: 16px;
+      padding-block-start: 2px;
+    }
+
+    .occurrences li {
+      font-family: monospace;
+      font-size: 12px;
+      cursor: pointer;
+    }
+
+    .occurrences .node-name {
+      color: #0074e8;
+    }
+
+    .occurrences .node-id,
+    .occurrences .node-class {
+      color: #dd00a9;
+    }
+
+    .occurrences.collapsed ul {
```

Comment:
If using a `<details>` element, this collapsed state won't be needed. The contents outside the `<summary>` element will be hidden automatically.